conjur/client.py

```diff
@@ -7,10 +7,12 @@
 the Conjur server
 """
 
+# Builtins
 import logging
 
-from .api import Api
-from .config import Config as ApiConfig
+# Internals
+from conjur.api import Api
+from conjur.config import Config as ApiConfig
```

Comment:
Non relative imports will probably break local CLI testing use case

Reply:
What do you mean? I ran all unit tests and integrations tests on my local machine and worked.

Reply:
If you use `conjur.something` you might be testing with a pre-installed conjur module code for these two imports rather than your local checkout. using a relative import ensures that you always are testing with the appropriate codebase for what the code is running under.

Reply:
@tzheleznyak can you explain why you suggested absolute paths here? These are two separate opinions here